src/snapshot/metadata.rs

```diff
@@ -55,15 +57,6 @@ pub struct InstanceMetadata {
 }
 
 impl InstanceMetadata {
-    pub fn new() -> Self {
-        Self {
-            ignore_unknown_instances: false,
-            instigating_source: None,
-            relevant_paths: Vec::new(),
-            context: InstanceContext::default(),
-        }
-    }
-
```

Comment:
Standard Rust convention is to use `new` and also implement `Default`. It's something Clippy suggests and is generally preferred to use `::new` over `::default`.

Reply:
Interesting, I didn't get a clippy warn!
Thanks for this - I'll do so :)